File: Code Chef/JavaScript/800-899/823 Equalizing Numbers EQLZING.js
let input = '';
process.stdin.on('data', (data) => input += data);

const strToInt = (strArray) => {
    strArray.shift();
    const intArray = strArray.map((strItem) => strItem.trim().split(' ').map(Number));
    
    return intArray;
};

const canEqualizeNums = (num1, num2) => {
    const diff = Math.abs(num1 - num2);
    
    ((diff % 2) === 0)
        ? console.log('Yes')
        : console.log('No');
};

process.stdin.on('end', () => {
    const testcases = strToInt(input.trim().split('\n'));
    
    testcases.forEach((testcase) => {
        const [num1, num2] = testcase;
        canEqualizeNums(num1, num2);
    });
});

File: Code Chef/C Programming/800-899/823 Equalizing Numbers EQLZING.c
#include <stdio.h>
int main(void)
{
	int t,a,b;
	scanf("%d",&t);
	for(int i=0;i<t;i++)
	{
	    scanf("%d%d",&a,&b);
	    if(a==b)
	    {
	        printf("Yes\n");
	    }
	    else
	    {
	        int diff=a-b;
	        if(diff<0)
	        {
	            diff=-1*(diff);
	        }
	        if(diff%2==0)
	        {
	            printf("Yes\n");
	        }
	        else
	        {
	            printf("No\n");
	        }
	    }
	}
	return 0;
}
